php-practice_1.php

```diff
@@ -1,43 +1,130 @@
 <?php
 
+// Q1
+
+$name ='畠山';
+echo "私の名前は $name です";
 
 // Q2 四則演算
+$num = (5*4);
 
+echo ($num) ;
 
-// Q3 日付操作
+echo "\n";
+
+echo ($num / 2 );
 
+// Q3 日付操作
+$today = date ("y年 m月 d日 H時:i分:s秒");
 
+echo "現在時刻は、 $today です。";
 
 // Q4 条件分岐-1 if文
-
+$device = 'mac';
+
+if ($device === 'windows') {
+    
+    echo'使用OSは、windowsです。';
+}
+if ($device === 'mac') {
+    
+    echo'使用OSは、macです';
+    
+}else {
+        echo'どちらでもありません。';
+    }
 
 // Q5 条件分岐-2 三項演算子
+$age = 5;
+
+$message = ($age > 18) ? '成人です':'未成年です';
 
+echo $message;
 // Q6 配列
+$sitys = ['群馬','茨城','埼玉','栃木','千葉','東京','神奈川'];
 
+echo " $sitys[3]と$sitys[4]は関東地方の都道府県です。";
 
 // Q7 連想配列-1
-
+$sitys = [
+  '東京' =>'新宿区',
+  '神奈川' =>'横浜市',
+  '千葉' =>'千葉市',
+  '埼玉' =>'さいたま市',
+  '栃木' =>'宇都宮市',
+  '群馬' =>'前橋市',
+  '茨城' =>'水戸市',
+  ];
+
+  foreach ($sitys as $keys => $valuses){
+  echo $valuses ."\n";
+}
 
 // Q8 連想配列-2
-
+if(array_key_exists('埼玉',$sitys)===true){
+  echo '埼玉県の県庁所在地は、さいたま市です';
+}
 // Q9 連想配列-3
+$sitys ['北海道'] = '札幌市';
+$sitys ['沖縄'] = '那覇市';
 
+foreach ($sities as $key => $value){
+  //echo  $key. 'の県庁所在地は'. $value. 'です。'."\n";
+if($key==='東京' || $key==='千葉'|| $key==='神奈川'||$key==='埼玉'||$key==='群馬'||$key==='茨城'||$key==='栃木'){
+  echo $key. 'の県庁所在地は'. $value. 'です。'."\n";
+}else {
+  echo  $key . 'の県庁所在地は関東じゃない'."\n";
+}}
 
 
 // Q10 関数-1
 
-
+function sayHi($name){
+  echo $name .'こんにちは'."\n";
+}
```

Comment:
`{}`は改行しましょう！
また`.`の前後は空白を設けていただきたいです！
```suggestion
function sayHi($name)
{
  echo $name . 'こんにちは' . "\n";
}
```

Reply:
{}は開業して . は空白を設けました。